### PR TITLE
Added keystore config steps to Vulnerability detection and to Upgradi…

### PR DESCRIPTION
**Pull request by aritosteles**
## Description

Added keystore configuration steps to
- configuring-scans.rst
- upgrading-central-components.rst

### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

Configuring vulnerability detection:
image

Upgrading central components:
image

### CODE REVIEW

**Review by ncvicchi — APPROVED** [review]
LGTM!

**Review by tdrauncieal — APPROVED** [review]
LGTM.

File: source/user-manual/capabilities/vulnerability-detection/configuring-scans.rst [hunk 1]
```diff
@@ -55,6 +55,13 @@ The following configuration block shows a configuration example for Syscollector
       </synchronization>
     </wodle>
 
+Make sure you have saved the Wazuh indexer username and password into the Wazuh manager keystore using the Wazuh-keystore tool.
```
Comment (javimed) [inline source/user-manual/capabilities/vulnerability-detection/configuring-scans.rst:58]:
```suggestion
You must save the Wazuh indexer username and password into the Wazuh manager keystore using the Wazuh-keystore tool.
```

File: source/user-manual/capabilities/vulnerability-detection/configuring-scans.rst [hunk 2]
```diff
@@ -55,6 +55,13 @@ The following configuration block shows a configuration example for Syscollector
...
+
+.. code-block:: console
+  
+  # /var/ossec/bin/wazuh-keystore -f indexer -k username -v <INDEXER_USERNAME>
```
Comment (javimed) [inline source/user-manual/capabilities/vulnerability-detection/configuring-scans.rst:62]:
```suggestion
   # /var/ossec/bin/wazuh-keystore -f indexer -k username -v <INDEXER_USERNAME>
```

File: source/user-manual/capabilities/vulnerability-detection/configuring-scans.rst [hunk 3]
```diff
@@ -55,6 +55,13 @@ The following configuration block shows a configuration example for Syscollector
...
+  # /var/ossec/bin/wazuh-keystore -f indexer -k username -v <INDEXER_PASSWORD>
```
Comment (javimed) [inline source/user-manual/capabilities/vulnerability-detection/configuring-scans.rst:63]:
```suggestion
   # /var/ossec/bin/wazuh-keystore -f indexer -k username -v <INDEXER_PASSWORD>
```

**Review by javimed — CHANGES_REQUESTED** [review]
Are we ok with the steps order? Just mentioning. In the installation guide we have the step before the ossec.conf edition step.

https://github.com/wazuh/wazuh-documentation/blob/v4.8.0-beta6/source/installation-guide/wazuh-server/step-by-step.rst?plain=1#L78-L88

File: source/user-manual/capabilities/vulnerability-detection/configuring-scans.rst (hunk 1 above)
Comment (ncvicchi) [reply source/user-manual/capabilities/vulnerability-detection/configuring-scans.rst:58]: There is no precedence or mandatory order, as long as both changes are applied before the Manager is restarted

**Review by javimed — APPROVED** [review]

**Pull request merged**